paddle/phi/infermeta/nullary.cc

```diff
@@ -115,4 +115,27 @@ void TruncatedGaussianRandomInferMeta(const std::vector<int>& shape,
   out->set_layout(DataLayout::NCHW);
 }
 
+void TrilIndicesInferMeta(
+    int rows, int cols, int offset, DataType dtype, MetaTensor* out) {
+  // number of elements in the first row of the tril,bounded by [0, cols]
+  auto m_first_row =
```

Comment:
use `n_first_row` and `n_last_row` is better?  the form of variable name is unified with `n_row_all` and `n_row_trapezoid` below

Reply:
done